pravega/pravega
# Issue 6505: Tweaking the use of the cluster name property in the Admin CLI

Signed-off-by: Kuldeep Kumar <kuldeep.kumar3@dell.com>

**Change log description**  
This PR allows a consistent use of pravegaservice.cluster.name by Admin CLI cluster commands and other commands (ie, bk list)

**Purpose of the change**  
Fixes https://github.com/pravega/pravega/issues/6505

**What the code does**  
Remove the hardcoded cluster namespace prefix, making it fully overridable via pravegaservice.cluster.name

**How to verify it**  
The following affected commands are working as expected.
In order to see the expected result user need to set `pravegaservice.clusterName`to `pravega/<clustername>`

    cluster get-host-by-container
    cluster list-instances
    cluster list-containers
